Assets/VRTK/Scripts/VRTK_InteractTouch.cs

```diff
@@ -401,6 +401,7 @@ private void CreateTouchCollider()
                 {
                     controllerCollisionDetector = Instantiate(customRigidbodyObject, transform.position, transform.rotation) as GameObject;
                     controllerCollisionDetector.transform.SetParent(transform);
+                    controllerCollisionDetector.transform.localScale = transform.localScale;
```

Comment:
It scales it down. I guess it depends on how you set it up. If you think of it like the controllers that auto scale up and down with the camera scale then yes you'd want to. If you want it to stay the same size no matter what scale the CameraRig is then no. Personally I think it makes sense to do it.

I was thinking of adding this 
```controllerCollisionDetector = Instantiate(customRigidbodyObject, transform.position, transform.rotation) as GameObject;
                    Vector3 originalScale = controllerCollisionDetector.transform.localScale;
                    controllerCollisionDetector.transform.SetParent(transform);
                    controllerCollisionDetector.transform.localScale = originalScale;```